modin/tests/pandas/test_series.py

```diff
@@ -1387,6 +1387,70 @@ def test_constructor_arrow_extension_array():
     df_equals(md_ser.dtypes, pd_ser.dtypes)
 
 
+def test_pyarrow_constructor():
+    pa = pytest.importorskip("pyarrow")
+    data = list("abcd")
+    df_equals(*create_test_series(data, dtype="string[pyarrow]"))
+    df_equals(*create_test_series(data, dtype=pd.ArrowDtype(pa.string())))
+
+    data = [["hello"], ["there"]]
+    list_str_type = pa.list_(pa.string())
+    df_equals(*create_test_series(data, dtype=pd.ArrowDtype(list_str_type)))
+
+
+def test_pyarrow_functions():
+    pytest.importorskip("pyarrow")
+    modin_series, pandas_series = create_test_series(
+        [-1.545, 0.211, None], dtype="float32[pyarrow]"
+    )
+    df_equals(modin_series.mean(), pandas_series.mean())
```

Comment:
eval_general?

Reply:
`mean()` returns a floating point number, so I don't see the need for it